functions/src/screenName.ts

```diff
@@ -0,0 +1,135 @@
+import { DocumentData, FieldPath } from "firebase-admin/firestore";
+import * as functions from "firebase-functions";
+import { db } from "./index";
+
+export const ScreenName = functions.https.onRequest(async (req, res) => {
+	//getting variables from request and checking them
+	if (req.method == "GET") {
+		var name: any = req.query.name as string;
+		var precision: any = req.query.precision;
+	} else if (req.method == "POST") {
+		var name: any = req.body.name as string;
+		var precision = req.body.precision;
+	}
+	if (!name|| name.length < 7) {
+		res.status(400).send('Parameter "name" - min 7 characters');
+		return;
+	}else if (!precision || precision < 0.9 || precision > 1) {
+		res.status(400).send('Parameter 0.9 < "precison" < 1');
+		return;
+	}
```

Comment:
Let's remove the limit of precision now that we have the limit in place.

Reply:
No limit no more